Dockerfile

```
@@ -30,11 +30,18 @@ RUN apt-get update && apt-get install -y \
   automake \
   autoconf \
   libtool \
-  protobuf-compiler \
   libprotobuf-dev \
+  unzip \
   --no-install-recommends && \
   rm -rf /var/lib/apt/lists/*
 
+# Install a more recent release of protoc (protobuf-compiler in jammy is 4 years old and misses some features)
+RUN cd /tmp && \
+    curl -sSL https://github.com/protocolbuffers/protobuf/releases/download/v25.2/protoc-25.2-linux-x86_64.zip -o protoc.zip && \
```

could we extract the version into the evars below?

```suggestion
    curl -sSL https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOC_VER}/protoc-${PROTOC_VER}-linux-x86_64.zip -o protoc.zip && \
```